examples/examples/core_client.rs

```diff
@@ -42,7 +42,7 @@ async fn main() -> anyhow::Result<()> {
 	let uri = Url::parse(&format!("ws://{}", addr))?;
 
 	let (tx, rx) = WsTransportClientBuilder::default().build(uri).await?;
-	let client: Client = ClientBuilder::default().build_with_tokio(tx, rx);
+	let client: Client<_> = ClientBuilder::default().build_with_tokio(tx, rx);
```

Comment:
Shame about the new generic; I guess the default param doesn't help here!

Reply:
it's fixed now, needless type hint